app/src/main/java/com/infomaniak/mail/ui/main/thread/SubjectFormatter.kt

```diff
@@ -97,20 +100,20 @@ class SubjectFormatter @Inject constructor(private val context: Context) {
     private fun postFixWithFolder(
         previousContent: CharSequence,
         folderName: String,
-        ellipsizeTag: EllipsizeConfiguration
+        ellipsizeConfiguration: EllipsizeConfiguration
     ) = context.postfixWithTag(
         previousContent,
         folderName,
-        TagColor(R.color.folderNameBackground, R.color.folderNameTextColor),
-        ellipsizeTag
+        TagColor(R.color.tagBackground, R.color.tagTextColor),
```

Comment:
These two resources should be named correctly, they're only used for folder name tags not all tags as their name implies